src/VelocityUKF.cpp

```diff
@@ -3,7 +3,7 @@
 using namespace uwv_kalman_filters;
 
 const std::string VelocityUKF::acceleration_measurement = "acceleration";
```

Comment:
Since all measurements are brought in the same form and transported by the same type they are distinguished by names. An enum doesn't work since the type cannot know of all the sorts of measurements the users want to have. It could also be solved by inheritance, but would force all users to implement their own types.